proto/attest/attest.proto

```diff
@@ -12,57 +12,57 @@ option java_package = "org.lfedge.eve.attest";
 // This is the request payload for POST /api/v2/edgeDevice/id/<uuid>/attest
 // The message is assumed to be protected by signing envelope
 message ZAttestReq {
-  ZAttestReqType reqType = 1;  //type of the request
-  ZAttestQuote quote = 2;      //attestation quote msg
-  repeated org.lfedge.eve.certs.ZCert certs = 3;    //X509 certs in .PEM format, signed by device certificate
-  AttestStorageKeys storage_keys = 4;    //encrypted secrets to be saved by the Controller, like encrypted keys for the volume storage vaults
+  ZAttestReqType reqType = 1;                    // type of the request
+  ZAttestQuote quote = 2;                        // attestation quote msg
+  repeated org.lfedge.eve.certs.ZCert certs = 3; // X509 certs in .PEM format, signed by device certificate
+  AttestStorageKeys storage_keys = 4;            // encrypted secrets to be saved by the Controller, like encrypted keys for the volume storage vaults
 }
 
 // This is the response payload for POST /api/v2/edgeDevice/id/<uuid>/attest
 // The message is assumed to be protected by signing envelope
 message ZAttestResponse {
-  ZAttestRespType respType = 1;    //type of the response
-  ZAttestNonceResp nonce = 2;      //nonce from Controller
-  ZAttestQuoteResp quoteResp = 3;  //attest quote response from Controller
-  AttestStorageKeysResp  storage_keys_resp = 4;   //attest storage_keys response from Controller
+  ZAttestRespType respType = 1;         // type of the response
+  ZAttestNonceResp nonce = 2;           // nonce from Controller
+  ZAttestQuoteResp quoteResp = 3;       // attest quote response from Controller, this can includes the encrypted keys
+  AttestKeysResp storage_keys_resp = 4; // attest storage keys response from Controller
 }
 
 enum ZAttestReqType {
   ATTEST_REQ_NONE = 0;
-  ATTEST_REQ_CERT = 1;   //EVE X.509 certificates
-  ATTEST_REQ_NONCE = 2;  //nonce request to Controller
-  ATTEST_REQ_QUOTE = 3;  //quote msg
-  Z_ATTEST_REQ_TYPE_STORE_KEYS = 4; //to store device keys in Controller
+  ATTEST_REQ_CERT = 1;        // EVE X.509 certificates
+  ATTEST_REQ_NONCE = 2;       // nonce request to Controller
+  ATTEST_REQ_QUOTE = 3;       // quote msg from device
+  Z_ATTEST_REQ_TYPE_KEYS = 4; // to store device keys in Controller
```

Comment:
Here STORE is a verb having nothing to do with storage, thus I think you should keep it.

Reply:
done.